draft-irtf-cfrg-opaque.md

```diff
@@ -1865,6 +1865,26 @@ response during registration as an oracle for whether a given client identity is
 registered. Applications should mitigate against this type of attack by rate
 limiting or otherwise restricting the registration flow.
 
+## Protecting the masking key during registration
+
+The user enumeration prevention method uses a symmetric encryption key generated
+by the client and sent to the server at registration in the clear. This relatively
+sensitive and secret piece of data requires the full registration to be secured
+with integrity, authenticity, and confidentiality, as offered by TLS, for example.
```

Comment:
I would remove "authenticity" because it is redundant, or if it really needs to be there, at least replace with "authentication"

Reply:
There's no authentication at the registration phase. The wording could be misleading.

Regarding authenticity, it's with regard to server messages really coming from the server.

Reply:
Does this answer the question?